### DISCUSSION TITLE
Missing documentation

### DISCUSSION THREAD

**Discussion by MyGIS** [GeoCat/qgis-bridge-plugin#157] [General]
Hello, I tried to access the documentation online from QGIS. I was forwarded to this link which id broken: https://geocat.github.io/qgis-bridge-plugin/v4.3.2/ 

404

File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.
[GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)

**Comment by GeoSander** [comment]
Which version are you using @MyGIS?
I believe we have fixed this some time ago (you can see that on the bottom of the About panel of the Publish dialog), or else it is a regression.

Anyway, the docs are here: https://geocat.github.io/qgis-bridge-plugin/v4.3/

**Comment by MyGIS** [comment]
QGIS version 3.28.1-Firenze
GeoCat Bridge Installed version 4.3.2
OS Windows 10

**Reply by GeoSander** [reply]
Yeah of course, your link already mentioned the version 🤦
Anyway, we're several minor versions further now, so please upgrade the plugin using the QGIS Plugin Manager.

**Comment by MyGIS** [comment]
The plugin is not upgradable at this point, perhaps it is because I need to update QGIS. Anyway, I just wanted to highlight this. This ticket can be closed for now.

**Reply by GeoSander** [reply]
No, 4.3.2 really is the last version.. I was reading 4.2.3 for some reason.
Guess I shouldn't respond to these things when I am on holiday... 😅

Marking this as a bug and I will fix it later...
